### ISSUE TITLE
In Scan mode, Narrator is passing the information for the scroll bar as just image.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/monaco-editor#1136]
**Blade Name:** Microsoft_Azure_IotHub > DeviceTwinBlade

**User Experience:**
User depending on screen reader might get affected as screen reader is passing the information for the scroller as just image which may confuse the user.

**Environment**
OS Build: Microsoft Windows version 1809 (OS Build 17758.1)
Browser: Edge (Microsoft Edge 44.17758.1.0)

**Prerequisite:**
Turn on narrator when in use

**Repro Steps:**
1. Navigate to 'All Services' and Search "IoT Hub" and select it.
2. Select the created IoT Hub from the list.
3. New screen opens. Now navigate to "IoT EDGE (Preview)" under Automatic Device Management from the left and select it.
4. New Blade "IoT EDGE (Preview)" Opens.
5. Navigate to "IoT EDGE Devices" Tab and select it.
6. Now select any IoT EDGE Device from the table. It will redirect user to the "Device Details" screen.
7. Again navigate to "Device Twin" button from the top and select it. New screen opens.
8. Navigate in scan mode and verify the information passed by screen reader for Scroll Bar.

**Actual Result:**
In Scan Mode, screen reader is passing the information for scrollbar as just image.

**Expected Result:**
In Scan mode, screen reader focus should not move to Scroll bar.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7ba12f3779-bdf1-40c2-bbcc-0e314ea94858%7d

![scrollbar](https://user-images.githubusercontent.com/30456910/47348827-ed250200-d6cf-11e8-9294-a18fe258d148.png)

**Comment by sonimishra** [comment]
We have found this issue while testing Azure IOT Hub, logging this issue as it belongs to Monaco team
Bug Reference Id: [Bug 2770295](https://msazure.visualstudio.com/One/_workitems/edit/2770295)

**Comment by alexdima** [comment]
I have investigated this in the context of #1666. Edge + Narrator fail to ignore `aria-hidden: true` and there is no way to let them know that it is just a presentation `<div>` which should not be hit by scan mode.

**Comment by alexdima** [comment]
Closing, as Microsoft Edge Legacy has reached end of support on March 9, 2021. See [the end of support for Microsoft Edge Legacy announcement](https://docs.microsoft.com/en-us/lifecycle/announcements/edge-legacy-eos-details)
